app/routes/circulars._archive._index/route.tsx

```diff
@@ -250,6 +251,12 @@ export default function () {
           Synonym Moderation
         </Link>
       )}
+      {queryFallback && (
+        <Hint id="searchFailedHint" style={{ color: 'red' }}>
+          "{query}" does not adhere to lucene search syntax. Please refer to the
```

Comment:
Please make sure that we are using consistent terminology. Do we call this "Lucene search syntax" (note capitalization) or "advanced search syntax?"